File: Java-Core/src/main/java/presto/parser/SParserFactory.java
package presto.parser;

import org.antlr.v4.runtime.ANTLRInputStream;


public class SParserFactory implements IParserFactory {

	@Override
	public ILexer newLexer() {
		return new SIndentingLexer(new ANTLRInputStream(""));
	}

	@Override
	public IParser newParser() {
		return new SCleverParser("");
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/parser/SParserFactory.js
var antlr4 = require("antlr4/index");
var SIndentingLexer = require("./SIndentingLexer").SIndentingLexer;
var SCleverParser = require("./SCleverParser").SCleverParser;

antlr4.SParserFactory = function() {
	
	this.newLexer = function(data) {
		return new SIndentingLexer(new antlr4.InputStream(data));
	};

	this.newParser = function(path, data) {
		return new SCleverParser(path, data);
	};

};
